docs/content/commands.md

```diff
@@ -70,34 +72,37 @@ public class BuildSystem : ComponentSystem
     {
         for(var i = 0; i < data.Length; i++)
         {
-            var requestSender = data.BuilderRequestSender[i];
+            var requestSender = data.BuildWallSender[i];
             var entityId = data.EntityIds[i];
             
-            BuildRequest buildRequest = new BuildRequest
+            Builder.BuildWall.Request request = new Builder.BuildWall.Request
             {
-                Location = new Location(...),
-                Rotation = new Rotation(...)
-            }
+                TargetEntityId = entityId,
+                Payload = new BuildRequest
+                {
+                    Location = new Location(...),
+                    Rotation = new Rotation(...)
+                }
+            };
+
             
-            requestSender.SendBuildWallRequest(entityId, buildRequest);
+            requestSender.RequestsToSend.Add(request);
         }
     }
 }
 ```
 
-This system runs on a client. It injects all SpatialOS entities in the client's view that have the `Builder` SpatialOS component, and sends a `build_wall` command to each SpatialOS entity (which will be received by the managed worker).
+This system is an example of sending command requests. Unity injects all ECS entities which have the `ShouldSendBuildWallCommand`, `Builder.CommandSenders.BuildWall`, and `SpatialEntityId` components into this system before `OnUpdate`. This system iterates over the injected entities and sends a build_wall command request to each of them.
 
```

Comment:
`build_wall`
(If we want to be consistent and super clear.)

Reply:
Woops.. nice catch :)